mappings/net/minecraft/entity/vehicle/AbstractMinecartEntity.mapping

```diff
@@ -14,7 +14,8 @@ CLASS axm net/minecraft/entity/vehicle/AbstractMinecartEntity
 		ARG 3 x
 		ARG 5 y
 		ARG 7 z
-	METHOD a setDamageWobbleStrength (F)V
+	METHOD a setWobbleMaxAngle (F)V
```

Comment:
This is a roll angle. Can we just call it a `roll`, which is more straightforward?

Reply:
I called it `angle` for consistency with the bubble column wobble, which is not a roll, pitch, or yaw (and therefore the only name I know for it is just `angle`). Also, what do you mean, exactly, by "call it a `roll`"? Do you mean `wobbleMaxRoll`? (Also, oops. Apparently I misremembered and called this a pitch before).

Reply:
This wobble is applied to the left-right roll of the minecart only, and this angle is only a roll. This is not a pitch (changed when going uphill/downhill) or a yaw (determines which direction minecart goes forward)

Reply:
If you don't have any qualms about the name, I'll go ahead and use it.